styleguide/source/_patterns/01-molecules/06-components/11-hub-card/_hub-card.scss

```diff
@@ -42,3 +42,31 @@
   margin: 0 24px;
   order: 2;
 }
+
+.hub-card-portrait {
+  min-height: 300px;
```

Comment:
but I don't think we want the height proportional to the view width? Why is that the basis for this? I guess I don't really understand what the math you are doing here is attempting.

IMO the min-height thing is a fallback that will rarely control the height as any block that is larger will "takeover".